libs/wire-api/test/golden/testObject_ListUsersById_2.json

```diff
@@ -0,0 +1,29 @@
+{
+    "failed": [],
```

Comment:
It's a bit unusual it appears in one case when there are no failed users, but not in another case when there are no failed users (the `testObject_ListUsersById_1.json` test case), but it's fine so long as it's been made clear to API users (i.e., client devs) this is an optional field and that its absence means there are no failed to list users.

Reply:
I've changed it over to a NonEmpty to have this encoded in the types, and removed this specific line as it can no longer be represented in the haskell.